.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 dist
 dist-ssr
 *.local
+package-lock.json
```

Comment:
```suggestion
```
We do want the `package-lock.json to be committed. You can install all the dependencies with `npm ci` which essentially does not try to resolve the dependency tree but just installs them in the versions defined in `package-lock.json`